### DISCUSSION TITLE
v0.6.0-M3

### DISCUSSION THREAD

**Discussion by erikerlandson** [erikerlandson/coulomb#306] [General]
- includes refactoring of `cats` support
- new priority based context rules, where applicable
- coulomb-testkit package for law checking

<hr /><em>This discussion was created from the release <a href='https://github.com/erikerlandson/coulomb/releases/tag/v0.6.0-M3'>v0.6.0-M3</a>.</em>

**Comment by erikerlandson** [comment]
@armanbilge  I cut a `v0.6.0-M3` release to publish `coulomb-testkit`. This also includes my priority-based `cats` refactoring.

**Reply by armanbilge** [reply]
Thanks, will give it a try soon!

**Reply by armanbilge** [reply]
Sorry, forgot to write an update here! We were able to update very easily, after changing the imports for Cats instances. Btw, we've already got a service in production on this :)
